crate2nix/Cargo.nix

```diff
@@ -2746,9 +2746,31 @@ rec {
           buildInputs = testInputs;
         } ''
           set -ex
-          cd ${crate.src}
+
+          export RUST_BACKTRACE=1
+
+          # recreate a file hierarchy as when running tests with cargo
+
+          # the source for test data
+          ${pkgs.xorg.lndir}/bin/lndir ${crate.src}
+
+          # build outputs
+          testRoot=target/debug
```

Comment:
What is the reasoning for calling it `target/debug`? To mimic how cargo does it?

Reply:
yes.